ONBOARDING.md

```diff
@@ -0,0 +1,84 @@
+- **Read (6 min)** [the OpenSearch CONTRIBUTING guide](https://github.com/opensearch-project/.github/blob/main/CONTRIBUTING.md)
+  - Please make sure you understand the implications of the [Developer Certificate of Origin](https://github.com/opensearch-project/.github/blob/main/CONTRIBUTING.md#developer-certificate-of-origin)
+
+### Resources and links
+
+There are more than 90 [GitHub repositories](https://github.com/opensearch-project/OpenSearch) that are part of the OpenSearch project. Explore the list to get a sense of the project breadth and scope.
+
+- The [roadmap](https://github.com/orgs/opensearch-project/projects/1) provides a single place to keep track of upcoming releases and current projects
+
+Looking for answers? While [the forum](https://forum.opensearch.org/) is always a good option, it’s worth checking these first
+
+- [Frequently Asked Questions](https://opensearch.org/faq)
+- [User documentation](https://opensearch.org/docs/latest/)
+- Developer documentation
+  - Start with `DEVELOPER_GUIDE.md` in the repository you’ll be working in (see [OpenSearch](https://github.com/opensearch-project/OpenSearch/blob/main/DEVELOPER_GUIDE.md) or [OpenSearch Dashboards](https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/DEVELOPER_GUIDE.md) guides)
+  - Most repositories have many other documentation files. To find them, search for Markdown files (`*.md` - [example](https://github.com/search?q=repo%3Aopensearch-project%2FOpenSearch-Dashboards+language%3AMarkdown&type=code&l=Markdown)), either on GitHub or in your editor.
+
+## Set up accounts and access
+
+### GitHub account
+
+To create issues, leave comments, or submit pull requests, you’ll need a GitHub account. Already have a GitHub account? Great! If not, it’s quick to get started
+
+1. **Sign up for an account** by navigating to https://github.com/ and following the prompts ([source](https://docs.github.com/en/get-started/onboarding/getting-started-with-your-github-account#1-creating-an-account)).
+2. **[Verify your email address](https://docs.github.com/en/get-started/signing-up-for-github/verifying-your-email-address)**
+3. Recommended - [**Set up GitHub SSH keys**](https://docs.github.com/en/authentication/connecting-to-github-with-ssh/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent)
+4. Recommended - **[Set up 2-factor authentication](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/about-two-factor-authentication)**
+5. Optional - If you’re new to using GitHub or git, we recommend reviewing “[Using GitHub's tools and processes](https://docs.github.com/en/get-started/onboarding/getting-started-with-your-github-account#part-2-using-githubs-tools-and-processes)”
+
+### OpenSearch forum account
+
+There are multiple ways to [connect with the OpenSearch community](https://opensearch.org/connect.html), but [the forums](https://forum.opensearch.org/) are the best place to start
```

Comment:
They are called "forum" on top and "forums" here; which one is it?

Reply:
We're not consistent and use both interchangeably in this repo: https://github.com/search?q=repo%3Aopensearch-project%2F.github%20forum&type=code
Same thing in the style guide: https://github.com/opensearch-project/documentation-website/blob/main/STYLE_GUIDE.md#brand-personality-traits

If it's up to me, I prefer the singular and will update accordingly.